src/Package/Impl/Publishing/Handlers/RmdPublishHandler.cs

```diff
@@ -28,7 +29,7 @@ internal sealed class RmdPublishHandler : IMarkdownFlavorPublishHandler {
             string outputFile = Path.GetFileName(outputFilePath);
             string outputFolder = Path.GetDirectoryName(outputFilePath).Replace('\\', '/');
             // Run rmarkdown::render
-            return $"\"rmarkdown::render(\'{inputFile}\', output_format=\'{format}\', output_file='{outputFile}',  output_dir='{outputFolder}')\"";
```

Comment:
If we control the encoding here, and this is not intended for direct user consumption, I wouldn't bother.